osmose_config.py

```diff
@@ -43,6 +43,7 @@ class template_config:
     dir_cache      = modules.config.dir_cache
     dir_scripts    = modules.config.dir_osmose
     bin_osmosis    = modules.config.bin_osmosis
+    bin_pyosmium_up_to_date = "pyosmium-up-to-date"
```

Comment:
Should it be moved to modules/config.py ?

Reply:
Done.